src/Umbraco.Core/Logging/DisposableTimer.cs

```diff
@@ -70,10 +77,11 @@ public class DisposableTimer : DisposableObjectSlim
                         }
                         else
                         {
-                            var args = new object[startMessageArgs.Length + 1];
+                            var args = new object[startMessageArgs.Length + 2];
```

Comment:
Is there a specific reason why you did not add the "IsEnabled" for the "Information" level?

Reply:
I was targeting debug level statements. Though in practice that check should be everywhere a log call is made, no matter the log level.